http/util.go

```diff
@@ -60,6 +60,10 @@ func rateLimitQuotaWrapping(handler http.Handler, core *vault.Core) http.Handler
 			return
 		}
 
+		for h, v := range quotaResp.Headers {
```

Comment:
IIRC in the RFC we had discussed having headers behind a config flag to prevent information leakage. Do we still want to do so?

Reply:
Sure, I'll add a config for this too.